packages/material-renderers/package.json

```diff
@@ -87,7 +87,7 @@
     "@emotion/styled": "^11.3.0",
     "@jsonforms/core": "3.1.1-alpha.0",
     "@jsonforms/react": "3.1.1-alpha.0",
-    "@mui/icons-material": "~5.11.16",
+    "@mui/icons-material": ">=5.11.16",
```

Comment:
`>=` seems a bit too lax to me because it also allows all higher major versions (e.g. 6.x, 7.x). I think we should change it to `^5.11.16` to allow all `5.x` versions but no other major versions. If it's known to work with 6.x it could then be adapted to `^5.11.16 || ^6.0.0`

Reply:
will do, thx

Reply:
it's done :D